pkg/agent/cniserver/server_windows_test.go

```diff
@@ -897,72 +898,86 @@ func TestReconcile(t *testing.T) {
 	}
 	kubeClient := fakeclientset.NewSimpleClientset(pods...)
 	cniServer.kubeClient = kubeClient
-	containerIfaces := map[string]*interfacestore.InterfaceConfig{
-		"iface1": {
-			InterfaceName: "iface1",
-			Type:          interfacestore.ContainerInterface,
-			OVSPortConfig: &interfacestore.OVSPortConfig{
-				PortUUID: generateUUID(t),
-				OFPort:   int32(3),
-			},
-			ContainerInterfaceConfig: &interfacestore.ContainerInterfaceConfig{
-				PodName:      "p1",
-				PodNamespace: testPodNamespace,
-				ContainerID:  generateUUID(t),
-			},
+	normalInterface := &interfacestore.InterfaceConfig{
```

Comment:
I didn't try to diff the interface definitions, but is there a significant difference between the Linux and Windows versions of the test interfaces, or could we use a shared definition in a common file?

Reply:
There should be no difference, I have moved them to common file.